package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "FECdemo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon ./server.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "morgan": "^1.9.0",
+    "express": "^4.16.2",
+    "react": "^16.2.0",
```

Comment:
Since you're pulled react and reactdom from the cdn, I don't think you need them listed as dependencies here